core/store/models/job_spec.go

```diff
@@ -216,7 +215,7 @@ type InitiatorParams struct {
 	Topics     Topics            `json:"topics,omitempty" gorm:"type:text"`
 
 	RequestData     JSON     `json:"requestData,omitempty" gorm:"type:text"`
-	Feeds           Feeds    `json:"feeds,omitempty" gorm:"type:text"`
+	Feeds           JSON     `json:"feeds,omitempty" gorm:"type:text"`
```

Comment:
Does this still work with old data? (Don't think we need a migration, just curious.)

Reply:
It *should*, although I haven't exactly tested it with old data sets... we used to save a json array of strings, and now we permit a json array of strings and objects.